code/src/pages/products/ProductsPage.test.tsx

```diff
@@ -3,34 +3,14 @@ import { fireEvent, screen } from "@testing-library/react";
 import ProductsPage from "@/pages/products/ProductsPage";
 import { useGetProductsQuery } from "@/store/api/productsApi";
 import { PaginationParams, RTKQueryReturnState } from "@/types/common";
+import { Product } from "@/types/product.types";
 import renderWithProviders from "@/utils/render-with-providers/renderWithProviders";
 
-jest.mock("@/hooks/use-snackbar/useSnackbar", () => ({
-  __esModule: true,
-  default: jest.fn(() => ({ openSnackbar: () => {} }))
-}));
-
-jest.mock("@/store/slices/userSlice", () => ({
-  __esModule: true,
-  default: () => ({}),
-  useUserDetailsSelector: jest.fn(() => ({ id: "123" })),
-  useIsAuthLoadingSelector: jest.fn(() => false)
-}));
-
-jest.mock("@/store/api/cartApi", () => ({
-  useAddToCartMutation: jest.fn(() => [jest.fn(), {}]),
-  useRemoveFromCartMutation: jest.fn(() => [jest.fn(), {}]),
-  useLazyGetCartItemsQuery: jest.fn(() => [jest.fn(), {}]),
-  endpoints: {
-    getCartItems: {
-      matchFulfilled: jest.fn()
-    }
-  }
-}));
-
-jest.mock("@/store/api/productsApi", () => ({
-  useGetProductsQuery: jest.fn()
-}));
+type ProductsContainerType = {
```

Comment:
repeated type here `ProductsContainerType` the same fields uses for `BestSellers` tests.
I suggest to create some common type and reuse it

Reply:
Thank for this suggestions! I've desided to re-use existing type